src/app/zap-templates/app-templates.json

```diff
@@ -41,11 +41,6 @@
         }
     ],
     "templates": [
-        {
-            "path": "templates/app/callbacks/PluginApplicationCallbacks.zapt",
```

Comment:
Shouldn't this file get removed?

Reply:
Yes, but I am unsure about the correct path.

In principle, since we do not use it, it should get removed. Arguing for the reverse: converting all of zap codegen to jinja/codegen.py is a large work item that if done correctly, not many see a difference (faster compiles, smaller repos, maybe easier to use templates ... sure, but app functionality would be identical). We have 111 (110 now) template files.

So my thinking is that "support build time codegen" is a worthwile change regardless if codegen.py or zap is used. As such, I need this removal to make sure zap does not codegen this file at all (so that compiles fail if I mess up my build time codegen) but I kept the template in case we want to say 'use zap for codegen after all'.

Then again ... the template can always be read back from source control history. I will remove it for completeness.